src/ruby/lib/grpc/generic/active_call.rb

```diff
@@ -59,7 +59,7 @@ class ActiveCall
     include Core::CallOps
     extend Forwardable
     attr_reader(:deadline)
-    def_delegators :@call, :cancel, :metadata, :write_flag, :write_flag=
+    def_delegators :@call, :cancel, :metadata, :write_flag, :write_flag=, :peer, :peer_cert
 
```

Comment:
I actually pushed a commit breaking that across two lines. I think that Jenkins build blew up for disk space reasons, or something like that. Should I push an empty commit to trigger a retry?